### PR TITLE
Updated README, commented code, submit blank uris

### PR DESCRIPTION
**Pull request by matthewrsj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,5 +24,8 @@ Activity: dev
 More activities? [y/n] n
 Time worked in minutes: 120
```
Comment (tschuy) [inline README.md]: Should change this as well

File: timesync [hunk 2]
```diff
@@ -12,33 +12,48 @@ import urllib.request
...
             activities.append(input('Activity: '))
 
+        # Duration in seconds - required
         duration = int(input('Time worked in seconds: '))
         issue_uri = input('Issue uri: ') or None
 
+        # Date Worked
         d = datetime.datetime.now()
+        # Month worked - optional
         month_worked = input('Month worked (format: 01): ')
```
Comment (tschuy) [inline timesync]: while you're here can you change these to be the `or d.month` thing I did with `issue_uri` above?

**Comment by matthewrsj** [comment]
Now produces this time when no uri provided:

```
{
    id: 21,
    duration: 18000,
    user: "mrsj",
    project: [
        "ts",
        "timesync"
    ],
    notes: "",
    issue_uri: null,
    date_worked: "2015-8-20",
    created_at: "2015-08-20",
    updated_at: null,
    activities: [
    "dev"
    ]
}
```

and when uri provided:

```
{
    id: 22,
    duration: 3600,
    user: "mrsj",
    project: [
        "ts",
        "timesync"
        ],
    notes: "",
    issue_uri: "https://github.com/tschuy/timesync-cli/pull/4/files",
    date_worked: "2015-8-20",
    created_at: "2015-08-20",
    updated_at: null,
    activities: [
    "dev"
    ]
}
```

**Comment by tschuy** [comment]
Looks good to me. Fixes #2

**Pull request merged**